sqldelight-studio-plugin/build.gradle

```diff
@@ -10,7 +10,7 @@ targetCompatibility = JavaVersion.VERSION_1_6
 
 intellij {
   updateSinceUntilBuild false
-  version 'IC-15.0.4'
+  version 'IC-143.870.1'
```

Comment:
unfortunately 15 has a breaking API change. Whenever Android Studio actually starts targeting it we can do the same but until then I think this has to be a 143 build